Sources/PointFree/Account/Account.swift

```diff
@@ -109,6 +103,34 @@ private let profileRowView = View<Database.User> { currentUser in
     ])
 }
 
+private let emailSettingCheckboxes = View<[Database.EmailSetting]> { currentEmailSettings in
+  [
+    p(["Receive email when:"]),
+    p([`class`([Class.padding([.mobile: [.left: 1]])])], Database.EmailSetting.Newsletter.allNewsletters.map { newsletter in
+      label([`class`([Class.display.block])], [
+        input(
+          [
+            type(.checkbox),
+            name("emailSettings[\(newsletter.rawValue)]"),
+            checked(currentEmailSettings.contains(where: ^\.newsletter == newsletter)),
+            `class`([Class.margin([.mobile: [.right: 1]])])
+          ]
+        ),
+        .text(encode(newsletterDescription(newsletter)))
+        ])
+    })
+  ]
+}
+
+private func newsletterDescription(_ type: Database.EmailSetting.Newsletter) -> String {
+  switch type {
+  case .announcements:
+    return "New announcements (very infrequently)"
+  case .newEpisode:
+    return "New episode is available (once a week)"
```

Comment:
lemme know if you have better copy for these...

Reply:
I think they're good! Maybe change "Receive email when:" to "Receive email:" for cohesion, tho.